dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/DqRuleInputEntryMapper.xml

```diff
@@ -40,4 +40,26 @@
         FROM t_ds_relation_rule_input_entry where rule_id = #{ruleId} )  b
         on a.id = b.rule_input_entry_id order by b.index
     </select>
+
+    <select id="getRuleInputEntryList" databaseId="pg" resultType="org.apache.dolphinscheduler.dao.entity.DqRuleInputEntry">
```

Comment:
Hi @eye-gu Although this is also a solution, but it means that we have to maintain two same sql.
Can this sql be splited into simple query? WDYT?

Reply:
Removing the backquote will cause unit testing to fail.

Reply:
Alternatively refer to Shenyu, which uses an Interceptor to replace backquotes with double quotes.
https://github.com/apache/shenyu/blob/master/shenyu-admin/src/main/java/org/apache/shenyu/admin/mybatis/pg/interceptor/PostgreSQLPrepareInterceptor.java

Reply:
WDYT? @zhongjiajie @ruanwenjun

Reply:
And there is another solution is to change this column name and totally avoid questions like this

Reply:
I  think the #14653 solution is better